skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -35,10 +35,7 @@ public function init() {
 	 * Use MW core variable to generate action button
 	 */
 	protected function prepareActionButton() {
-
-		global $wgTitle, $wgUser, $wgRequest;
-
-		$isDiff = !is_null( $wgRequest->getVal( 'diff' ) );
+		$isDiff = !is_null( $this->wg->Request->getVal( 'diff' ) );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)